### DISCUSSION TITLE
Changing the name of this project.

### DISCUSSION THREAD

**Discussion by anantnrg** [StrataWM/strata#16] [Polls]
Currently, the name is StrataWM, which imo seems a bit too generic. Also maybe its a bit long, something under 4 letters would do.

**Comment by vazw** [comment]
Sound cool, I would love to contributing to this projects to leverage my skills and extend my learning curve on Rust. but not sure which part should I start with. you're impressing me for a 14yo, keep going bro!.

**Reply by anantnrg** [reply]
Thanks! Currently me and @calops are working on implementing a Lua config for the compositor. I created a separate repo [here](https://github.com/anantnrg/lua_rust) to make it a little easier to work with. It uses `mlua` to execute the script. Also I've created a ROADMAP.md file in the repo. It contains what features to add and which are completed. Any further changes to the compositor would require looking into the source code of CosmicComp, a compositor written by the guys over at System76.  It contains a lot of features which need to be implemented here. 

Also currently I have my term exams starting on the 16th so I can only start working on this again after the 24th. But if you make any changes and create a pull request, I'll check it out. Glad to have you as a contributor here!

Cheers :)

**Comment by calops** [comment]
I don't mind "Strata" as a name, to be honest, but I would drop the "WM" part entirely (although it's fine to keep in the organization's name). Especially since its added value probably comes from the compositing features more than the WM ones (which are in general the easy part).

**Reply by anantnrg** [reply]
Hmm... I guess that makes sense. But I'd like to know what you think of ExoWM. I created an [issue here](https://github.com/stratawm/stratawm/issues/17). I also included a prototype logo for it. I personally think ExoWM sounds a bit better.

**Reply by calops** [reply]
I think "exo" is a fine prefix, but my issue with "WM" is that it's pretty much impossible to pronounce, at least not fluidly. Maybe there's something to be found by playing around the word "compositor" instead?

Nice logo though, although I slightly prefer the "S" you did for Strata.

**Reply by anantnrg** [reply]
ExoComp maybe? But on second thought I think Strata is better.

**Comment by anantnrg** [comment]
Well, I've decided to keep the Strata name but remove the "WM" part as @calops suggested.
